### ISSUE TITLE
[meta] Reccommend checking code with all clippy lints before filing a new issue

### ISSUE THREAD

**Issue by lolbinarycat** [rust-lang/rust-clippy#13424]
A lot of issues suggest adding lints that already exist.  This can lead to nasty experiences for new contributors when they realize they spent a week implementing something that already exists.  It's nearly impossible to look through all the hundreds of lints, but there's a little hack to see if the lint you want already exists:

```rust
//! Testing if a clippy lint already exists.

#![warn(clippy::pedantic)]
#![warn(clippy::nursery)]
#![warn(clippy::restriction)]
#![allow(clippy::blanket_clippy_restriction_lints)]

// insert your bad code here
```

just stick this into the playground and see what lints show up.  this could even be done automatically with a bot whenever a new lint is suggested, and show any lints that match as "possibly related".

**Comment by lolbinarycat** [comment]
I fixed this in https://github.com/rust-lang/rust-clippy/pull/15336 actually.
